docs/CNAME

```diff
@@ -0,0 +1 @@
+victory.formidable.com
```

Comment:
(The only reason victory.formidable.com already kind of works is that I pushed up this CNAME early for DNS troubleshooting)